packages/ag-charts-community/src/options/chart/chartOptions.ts

```diff
@@ -147,13 +147,15 @@ export interface AgBaseThemeableChartOptions<TDatum = any> {
     /** The height of the chart in pixels. */
     height?: PixelSize;
     /**
-     * By default (unset) the chart will assume a minimum size of 300x300 px, and grow with the container element.
-     *
-     * Set to true to remove the minimums (e.g. for use with a CSS-managed layout).
-     *
-     * Set to false to disable automatic sizing.
+     * Sets the minimum height of the chart.
+     * Default: `300`
+     */
+    minHeight?: PixelSize;
```

Comment:
Only thing about this naming is `height` overrides it. Maybe `preferredMinHeight` or something - or just adjust the documentation to mention height is an override

Reply:
I've adjusted the documentation, thanks!